compiler/rustc_ast_lowering/src/delegation.rs

```diff
@@ -224,8 +247,8 @@ impl<'hir> LoweringContext<'_, 'hir> {
             let mut parameters: Vec<hir::Param<'_>> = Vec::new();
             let mut args: Vec<hir::Expr<'hir>> = Vec::new();
 
-            for (idx, param_ty) in decl.inputs.iter().enumerate() {
-                let (param, pat_node_id) = this.generate_param(param_ty);
+            for idx in 0..param_count {
```

Comment:
Note to self: This also generates a param and an arg for `...` (`VaListImpl<'_>`) :+1:

Reply:
I've added a comment about this to `fn param_count`.